domains/auth/Auth.API/Tests/Services/OidcProviders/SignaturGruppenIT.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using API.Configuration;
+using API.Services;
+using API.Services.OidcProviders;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using RichardSzalay.MockHttp;
+using Xunit;
+using Xunit.Categories;
+
+namespace Tests.Services.OidcProviders;
+
+[UnitTest]
+public class SignaturGruppenIT
```

Comment:
```suggestion
public class TestSignaturGruppen
```

This will be using the same convention as the other tests in this Tests-project.

Reply:
Fixed